templates/web/base/admin/update_edit.html

```diff
@@ -55,10 +55,42 @@
 </li>
 <li>[% loc('Cobrand:') %] [% update.cobrand %]</li>
 <li>[% loc('Cobrand data:') %] [% update.cobrand_data OR loc('None') %]</li>
+[% IF update.external_id~%]
+  <li>[% loc('External ID') %]: [% update.external_id %] </li>
+[%~END %]
 <li>[% loc('Created:') %] [% PROCESS format_time time=update.created %]</li>
+<li>[% loc('Confirmed: ') ~%]
+  [%~ IF update.confirmed ~%]
+    [% PROCESS format_time time=update.confirmed %]</li>
+  [% ELSE ~%]
+    Not confirmed
+  [% END %]
 [% IF update.get_extra_metadata('external_status_code') %]
 <li>[% loc('External status code:') %] [% update.get_extra_metadata('external_status_code') | html %]</li>
 [% END %]
+<li>[% loc('Send state:') %] [% update.send_state %]</li>
+[% IF c.user.is_superuser AND update.send_fail_count > 0 AND update.send_state != 'sent' AND update.send_state != 'skipped' %]
+<ul>
+  <li>[% loc('Send fail count:') %] [% update.send_fail_count %]</li>
+  <li>[% loc('Send fail reason:') %] [% update.send_fail_reason %]</li>
+  <li>[% loc('Last send fail:') %] [% PROCESS format_time time=update.send_fail_timestamp %]</li>
+</ul>
+  [% IF update.send_state != 'sent' %]
+    <input type="submit" class="btn" name="mark_sent" value="[% loc('Mark as sent') %]">
+  [% END %]
+  [% IF update.send_state != 'skipped' %]
+    <input type="submit" class="btn" name="mark_skip" value="[% loc('Mark as skipped') %]">
+  [% END %]
+[% ELSIF update.send_state == 'sent' OR update.send_state == 'skipped' %]
```

Comment:
I think it'd be better if whensent is always shown (if it's set), rather than going on the send state - eg I think ones pulled in via Open131 will be processed but have a whensent (and possibly others).